system/jlib/jstatcodes.h

```diff
@@ -184,6 +184,8 @@ enum StatisticKind
     StNumAllocations,
     StNumAllocationScans,
     StNumDiskRetries,
+    StTimeGenerate,
+    StCycleGenerateCycles,
```

Comment:
@ghalliday is StCycleGenerateCyles generated anywhere?  (See code to generate StTimeGenerate but not StCycleGenerateCycles)